packages/@webex/contact-center/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@webex/contact-center",
```

Comment:
Should we call this `@webex/plugin-cc` to match the other publicly available plugins in the JS SDK? Same comment for the folder name also.

Reply:
Would it be alright to advertise CC package as plugin. We have been referring to it as CCSDK so I thought keeping plugin the name might not be the way to go?

Reply:
I'd like to get others to weigh in on this too. @arun3528 What do you think?

Reply:
As discussed over call, let's keep this as plugin-cc for now until we get confirmation.